### ISSUE TITLE
Error setting state

### ISSUE THREAD

**Issue by andrasg** [szokeptr/homebridge-tradfri-plugin#20]
Hi,
Every now and then I am experiencing unresponsive buttons from the Home app. When this happens a similar logitem is in the logs:

```
{ [Error: Command failed: coap-client -u 'Client_identity' -k '<PSK>' -B 5 -m PUT -e '{"3311":[{"5850":0}]}' coaps://192.168.x.x:5684/15001/65539
]
  killed: true,
  code: null,
  signal: 'SIGTERM',
  cmd: 'coap-client -u \'Client_identity\' -k \'<PSK>\' -B 5 -m PUT -e \'{"3311":[{"5850":0}]}\' coaps://192.168.x.x:5684/15001/65539' }
[8/27/2017, 11:50:47 PM] [Tradfri] Error setting state
```

Parallel to this, the Home app often shows No Response around the accessory names. How can I better diagnose this issue?

**Comment by andrasg** [comment]
I can consistently repro this when setting a scene from Homekit to a group of accessories (tradfri lights). The command seems to be sent to the lights almost instantaneously. Then there's a little delay (about 1 sec) till the error shows up and the Home app shows that the command was executed.

**Comment by szokeptr** [comment]
Hi, I published version 1.1.3 of the plugin, it should solve the scene setting stability issues. Please test and let me know, it works fine on my end now.
